fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonHlProjectUtils.xtend

```diff
@@ -1362,6 +1363,7 @@ class Test«cls.name»(object):
             assert tango_context.device.«attr.name» == «attr.defaultValueHL»
```

Comment:
This results in something like:
```python
        assert tango_context.device.healthState == HealthState.OK
```
which is good, but the `HealthState` class is not imported in the test class, so it would fail.
Adding `from BaseDevice import HealthState` after the other imports would help.

Reply:
I still don't see something like `from BaseDevice import HealthState` in the latest changes.

Reply:
I have added import statement in Device_test.py file for importing BaseDevice. Importing other Enum classes from the base device is a bigger activity and could be handled as a separate issue.

Reply:
OK, the import of the Enum classes for tests can be handled later.  It was already "broken" before this PR.

Reply:
Okay Anton. Thanks.